### PR TITLE
test(e2e,playwright): desktop layout

### PR DESCRIPTION
**Pull request by shabeebk**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Ref #51705

### CODE REVIEW

**Comment by shabeebk** [comment]
camperbot cant code

**Review by naomi-lgbt — APPROVED** [review]

File: e2e/desktop-layout-component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import { test, expect } from '@playwright/test';
+
+test.use({ storageState: 'playwright/.auth/certified-user.json' });
+
+test.describe('Classic challengs - 3 pane desktop layout component', () => {
```
Comment (raditotev) [inline e2e/desktop-layout-component.spec.ts:5]: typo

```diff
- challengs 
+ challenge 
```

File: e2e/desktop-layout-component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    const editorPane = desktopLayout.getByTestId('editor-pane');
+    await expect(editorPane).toBeVisible();
+
+    const previewPane = desktopLayout.getByTestId('preview-pane');
+    await expect(previewPane).toBeVisible();
+  });
+});
+
+test.describe('Classic challengs - 2 pane desktop layout component', () => {
```
Comment (raditotev) [inline e2e/desktop-layout-component.spec.ts:38]: Same typo `challengs`.

File: e2e/desktop-layout-component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
+import { test, expect } from '@playwright/test';
+
+test.use({ storageState: 'playwright/.auth/certified-user.json' });
+
+test.describe('Classic challengs - 3 pane desktop layout component', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(
+      'learn/2022/responsive-web-design/build-a-survey-form-project/build-a-survey-form'
+    );
+  });
```
Comment (raditotev) [inline e2e/desktop-layout-component.spec.ts:10]: There is a single test in this `describe` block, `beforeEach` hook is not really needed. It's ok if you decide  to leave it.

**Review by Ahmadkashif — APPROVED** [review]

File: e2e/desktop-layout-component.spec.ts (hunk 1 above)
Comment (shabeebk) [reply e2e/desktop-layout-component.spec.ts]: Fixed.

File: e2e/desktop-layout-component.spec.ts (hunk 2 above)
Comment (shabeebk) [reply e2e/desktop-layout-component.spec.ts]: Fixed.

File: e2e/desktop-layout-component.spec.ts (hunk 3 above)
Comment (shabeebk) [reply e2e/desktop-layout-component.spec.ts]: Removed the beforeEach hook with [825a9a9](https://github.com/freeCodeCamp/freeCodeCamp/pull/52012/commits/825a9a9c856598bfbf4b9232618dbec93f64967f)

**Comment by shabeebk** [comment]
Thanks, @raditotev for the comments. Changes made as needed.

Thanks for the approval @naomi-lgbt @Ahmadkashif 🥳

**Comment by raditotev** [comment]
Awesome 👍

**Pull request merged**